scripts/deploy.sh

```diff
@@ -71,15 +81,21 @@ elif [[ ! -z "$dtag" && -z "$ptag" && -z "$rtag" ]]; then
         exit 1
     fi
 
-    # tag it locally
+    # Merge branch into master and push to origin
+    git checkout master
+    git pull origin
+    git merge --no-ff release_$rtag -m "Merge branch devel_$rtag"
+    git push origin master
+    git branch -d release_$rtag
+
+    # Tag the version locally.
     git tag -a $dtag -m "New devel[rc] build $dtag."
+    git push origin $rtag
 
-    # and push the tag
-    git push origin $dtag
     echo "The new devel build was triggered."
 
 else
-    echo "You have to pass a -d tag (dev build) OR -p and -r tags (for releases)."
+    echo "You must pass a -d tag (dev build) OR -r tags (for releases) and a -p tag (previous version)."
```

Comment:
I think this would read more clearly (paraphrased): "a -d tag and a -p tag OR a -r tag and -p tag" Basically with the giant `OR` it doesn't seem like the "and a -p tag" distributes over both clauses.